src/main/java/core/basesyntax/model/Bet.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class Bet {
+    private int userId;
+    private int value;
+    private double risk;
+
+    public Bet() {
+    }
```

Comment:
I think we do not need constructor without parameters for Bet. Because it is no sense to create Bet without value and risk. And if we give such possibility to another developers they can make mistakes in future :)

Reply:
But I have ID so I could activate this somewhere right before inputting in db. But ok fixed.